src/components/VFilters/VSearchGridFilter.vue

```diff
@@ -1,8 +1,5 @@
 <template>
-  <div
-    class="filters py-8 px-10 min-h-full md:bg-dark-charcoal-06"
-    data-testid="filters-list"
-  >
+  <div class="filters py-8 px-10" data-testid="filters-list">
```

Comment:
Styles specific to the search filter _sidebar_ have been moved up in the component tree.